### PR TITLE
Http my branch

### PR DESCRIPTION
**Pull request by Bereketmebrahtu**
have a quick look and let me know what you think

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,10 +5,21 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
+    @import url('https://fonts.googleapis.com/css2?family=Bungee+Inline&display=swap');
```
Comment (PJSalter) [inline index.html:8]: Not sure if this is needed in both the html and css, maybe the link bar goes in here, however we can test that later,

Great work 🌟

File: index.html [hunk 2]
```diff
@@ -5,10 +5,21 @@
...
     <title>http-app-project</title>
 </head>
 <body>
+<section class="form-container">
```
Comment (PJSalter) [inline index.html:12]: great structuring in form layout, the class name defines what the container will keep inside.

good start on button for soon to be displayed giphy ✨

File: styles.css [hunk 3]
```diff
@@ -1 +1,3 @@
-/* ---------------- styling for App --------------------- */
\ No newline at end of file
+/* ---------------- styling for App --------------------- */
+
+@import url('https://fonts.googleapis.com/css2?family=Bungee+Inline&display=swap');
```
Comment (PJSalter) [inline styles.css:3]: really like the font we've come up with, brilliant work ✨

File: index.html (hunk 1 above)
Comment (Bereketmebrahtu) [reply index.html:8]: I agree, I think you only need it to go either in the head of the HTML or import through CSS.

**Pull request merged**